packages/dcos-metrics-collector/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+export GOPATH=$GOPATH:/pkg
+mkdir -p /pkg/src/github.com/dcos/dcos-metrics
+# Create the GOPATH for the go tool to work properly
+cp -R /pkg/src/* /pkg/src/github.com/dcos/dcos-metrics/
```

Comment:
Heres how you can do this:
https://github.com/dcos/dcos/blob/master/packages/3dt/build#L5